src/main/java/com/google/codeu/servlets/UserListServlet.java

```diff
@@ -3,7 +3,6 @@
 import com.google.codeu.data.Datastore;
 import com.google.gson.Gson;
 import java.io.IOException;
-
```

Comment:
sometimes the whitespace is nice when separating between imports from different kinds of packages (i.e. google vs java), but I don't quite recall what the convention for the rest of this project is.